vpdq/python/test_vpdq_faiss.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
```

Comment:
Yay tests!

Instead of the unittest framework @Dcallies is recommending we use pytest (and switch over the existing unittest)

Here an example PR he merged: https://github.com/facebook/ThreatExchange/pull/1029